### PR TITLE
Change lat long data type to decimal + add lat long to anchors

### PR DESCRIPTION
**Pull request by yafonia**

### CODE REVIEW

File: PetaNetra Map Editor/Utilities/GlobalVariables.swift [hunk 1]
```diff
@@ -29,11 +29,11 @@ struct GlobalVariable {
     static var worldMapURL: URL?
     
 }
-func joinLatLong(lat: Double, long: Double) -> String {
-    let latSplit = String(lat).split(separator: ".")
+func joinLatLong(lat: Decimal, long: Decimal) -> String {
```
Comment (alifmahardhika) [inline PetaNetra Map Editor/Utilities/GlobalVariables.swift:32]: this function results in 3 decimal point string representation.
eg:
lat = 101.12345678
long = 90.12345678

result = 101.123,90.123.

used in arworldmap naming. i think should use 5 dec point minimal or just concat the lat,long without changes to decimal point

File: PetaNetra Map Editor/Utilities/Networking/NetworkingUtils.swift [hunk 2]
```diff
@@ -29,8 +29,8 @@ class NetworkingUtils {
     }
     
     //Deprecatednya pake yang ini
-    func shortenCoordinateReturnString(coord: Double) -> String {
-        let a = String(coord)
+    func shortenCoordinateReturnString(coord: Decimal) -> String {
```
Comment (alifmahardhika) [inline PetaNetra Map Editor/Utilities/Networking/NetworkingUtils.swift:32]: 3 point decimal too

File: PetaNetra Map Editor/Utilities/Networking/NetworkingUtils.swift [hunk 3]
```diff
@@ -39,13 +39,13 @@ class NetworkingUtils {
     }
     
     //Deprecated
-    func shortenCoordinate(coord: Double) -> Double {
-        let a = String(coord)
+    func shortenCoordinate(coord: Decimal) -> Decimal {
```
Comment (alifmahardhika) [inline PetaNetra Map Editor/Utilities/Networking/NetworkingUtils.swift:42]: same

File: PetaNetra Map Editor/Services/Networking/UploadServices.swift [hunk 4]
```diff
@@ -205,8 +205,8 @@ final class UploadServices {
         }
     }
     
-    func shortenCoordinateReturnString(coord: Double) -> String {
-        let a = String(coord)
+    func shortenCoordinateReturnString(coord: Decimal) -> String {
```
Comment (alifmahardhika) [inline PetaNetra Map Editor/Services/Networking/UploadServices.swift:208]: same

**Review by alifmahardhika — CHANGES_REQUESTED** [review]
update 3 point decimal uses to minimal 5 point atau gausa dipotong juga ndapapa

**Comment by yafonia** [comment]
changed from 3 to 5 dec point. thanks alif!

**Review by alifmahardhika — APPROVED** [review]

**Pull request merged**